docs/app/Views/Usage.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import pkg from 'package.json'
+import {
+  Container,
+  Header,
+  Segment,
+} from 'src'
+import Logo from '../Components/Logo/Logo'
+import { semanticUIDocsURL, semanticUIRepoURL, semanticUICSSRepoURL } from 'docs/app/utils'
+
+const Usage = () => (
+  <Container id='usage-page'>
+    <Segment basic textAlign='center'>
+      <Logo centered size='small' />
+      <Header as='h1' textAlign='center'>
+        Semantic-UI-React
+        <Header.Subheader>
+          {pkg.description}
+        </Header.Subheader>
+      </Header>
+    </Segment>
+
+    <Segment basic padded>
+      <Header as='h2' dividing>Usage</Header>
+      <p>
+        The Semantic UI React package can be installed via NPM:
+      </p>
+      <Segment>
+        <pre>$ npm install {pkg.name}</pre>
```

Comment:
Can we add the `--save` flag here to remind users to persist the install to their deps?

Since this new page exists, we can also remove the installation instructions from the Introduction page.  Do you think we should replace that one with a `Setup` section that just links to this `Usage` section?